packages/svelte-form/eslint.config.js

```diff
@@ -0,0 +1,5 @@
+// @ts-check
+
+import rootConfig from '../../eslint.config.js'
+
+export default [...rootConfig]
```

Comment:
do you think there's opportunity here to enable svelte rules for the svelte files in here?

not in this PR, but we could open an issue to do it separately if you think that makes sense

Reply:
honestly I don't care, Eslint isn't of much use in this repo in my opinion, so I'd rather just keep it as is

Reply:
to be clear, we were always going to keep it as it is _in this pr_. i was asking in case its worth us opening an issue to do it in a separate PR

> not in this PR, but we could open an issue to do it separately